docker/Dockerfile

```
@@ -0,0 +1,76 @@
+FROM rocker/tidyverse:3.6.1
+MAINTAINER ccdl@alexslemonade.org
+WORKDIR /rocker-build/
+
+RUN apt-get update && apt-get install -y --no-install-recommends apt-utils
+RUN apt-get install dialog apt-utils -y
+
+RUN apt-get update -qq && apt-get -y --no-install-recommends install \
+    build-essential \
+    libxml2-dev \
+    libsqlite-dev \
+    libmariadbd-dev \
+    libmariadbclient-dev \
+    libmariadb-client-lgpl-dev \
+    libpq-dev \
+    libssh2-1-dev \
+    pandoc \
+    libmagick++-dev \
+    time
+
+RUN install2.r \
+    rjson \ 
+    ggpubr \
+    colorspace\
+    Rtsne \
+    umap
+
+ENV BIOCONDUCTOR_VERSION 3.10
+RUN R -e "BiocManager::install(c('ensembldb', 'DESeq2', 'qvalue',\
+    'org.Hs.eg.db', 'org.Dr.eg.db', 'org.Mm.eg.db', 'org.Cf.eg.db',\
+    'ComplexHeatmap', 'ConsensusClusterPlus', 'scran', 'scater',\
+    'tximport', 'alevinQC'), \
+    version = '${BIOCONDUCTOR_VERSION}', update = FALSE)"
+
+# FastQC
+RUN apt update && apt install -y fastqc
+
+# fastp 
+RUN git clone https://github.com/OpenGene/fastp.git
+RUN cd fastp && \
+    make && \
+    sudo make install
+
+# MultiQC
+RUN apt update && apt install -y python-pip
+RUN pip install multiqc
```

Super minor, but I switched to python3 for this on the server.
```suggestion
RUN apt update && apt install -y python3-pip
RUN pip3 install multiqc
```